### PR TITLE
chore(ci): set a shorter cache on manifest

### PR DESCRIPTION
**Pull request by binoy14**
### Description

Update the cache header on the manifest to be 10s and JS files to 1 year immutable

FIXES SDX-1343

### What to review

Changes makes sense

### Testing

N/A

### Notes for release

N/A (internal)

### CODE REVIEW

**Comment by binoy14** [comment]
* **#6660** <a href="https://app.graphite.dev/github/pr/sanity-io/sanity/6660?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `next`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @binoy14 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by bjoerge** [comment]
Needs a rebase?

File: scripts/uploadBundles.ts [hunk 1]
```diff
@@ -160,6 +160,10 @@ async function updateManifest(newVersions: Map<string, string>) {
     const options = {
       destination: 'modules/v1/manifest-v1.json',
       contentType: 'application/json',
+      metadata: {
+        // 10 mins cache
+        cacheControl: 'public, max-age=600',
```
[the remark this answers is not in the document]
Comment (binoy14) [reply scripts/uploadBundles.ts:165]: What if the manifest is only used on the server? In that case won't it just be the (600) from the cdn?

[the remark this answers is not in the document]
Comment (sgulseth) [reply scripts/uploadBundles.ts:165]: Depends on how many proxies and caches are in the chain, but if it's just a http call to a gcp bucket it _should_ only be up to 600s

[the remark this answers is not in the document]
Comment (binoy14) [reply scripts/uploadBundles.ts:165]: yep that is what it is, I'll DM you to confirm. Thanks for jumping in

**Review by rexxars — APPROVED** [review]
👍

**Pull request merged**